compiler/rustc_borrowck/src/region_infer/values.rs

```diff
@@ -114,15 +116,15 @@ crate enum RegionElement {
 /// points for each region-vid.
```

Comment:
Maybe update this doc comment to mention "interval matrix" instead of "bit matrix" ?